pr_4/javascript/random_samples/nosql_injection.js

```
@@ -0,0 +1,49 @@
+            } else {
+                data.Items.forEach(function(element, index, array) {
+                    console.log(element.Title.S + " (" + element.Subtitle.S + ")")
+                })
+            }
+       })
+    })
+}
+// {/fact}
+
+// {fact rule=nosql-injection@v1.0 defects=0}
+var AWS = require('aws-sdk')
```

Biome (2.1.2)

[error] Shouldn't redeclare 'AWS'. Consider to delete it or rename it.

'AWS' is defined here:


(lint/suspicious/noRedeclare)